docs/reference/commandline/image_prune.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "image prune"
+description = "Remove all stopped images"
+keywords = [image, prune, delete]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# image prune
+
+```markdown
+Usage:	docker image prune
+
+Delete unused images
+
+Options:
+  -a, --aggressive   Delete all unused images not just dangling ones
```

Comment:
I think we call this `all` on other commands, do you think the behaviour here is different enough to change the name?